conf/application.conf

```diff
@@ -153,6 +153,7 @@ assets {
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:6791/customs-declare-exports/start"
+  firstPageInService = "/customs-declare-exports/choice"
```

Comment:
Since these two urls are related to this service, there is a better way to define them. You can simply put `controllers.routes.ChoiceController.displayPage().url` in any place you need to redirect to the Choice Page.